### PR TITLE
Multirate Extended Kalman Filter for Omnidirectional Robot

### PR DESCRIPTION
**Pull request by acamalinovic**
This is matlab code for Omni Robot .

### CODE REVIEW

**Comment by alexsb92** [comment]
I don't think you were supposed to merge it just yet.
What should I do ?

File: matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/omniRobotFrame.m [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function [Xideal,X,Y,V]=omniRobotFrame(r,l,W,Xinit,dt,e)
+
```
Comment (ajaysinghchhonker) [inline matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/omniRobotFrame.m:2]: Please add few comments, what are W and V used for?

File: matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/omniRobotFrame.m~ [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function [Xideal,X,Y,V]=omniRobotFrame(r,l,W,X0,dt,e)
+
```
Comment (acamalinovic) [inline matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/omniRobotFrame.m~:2]: % W IS VEKTOR OF THE WHEELS VELOCITIES W=[W1, W2, W3]
% V IS VELOCITY OF THE OMNIDIRECTIONAL ROBOT

File: matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/omniRobotFrame.m (hunk 1 above)
Comment (acamalinovic) [inline matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/omniRobotFrame.m:2]: W is vector of the wheels velocity W1 ,W2, W3
V is final velocity of the robot.

**Pull request merged**